internal/worker/sshsession/worker_whitebox_test.go

```diff
@@ -0,0 +1,156 @@
+package sshsession
+
+import (
+	"os"
+
+	"github.com/juju/loggo"
+	"github.com/juju/testing"
+	gc "gopkg.in/check.v1"
+)
+
+type workerWhiteboxSuite struct {
+	testing.IsolationSuite
+}
+
+var (
+	_                  = gc.Suite(&workerWhiteboxSuite{})
+	sshdConfigTemplate = `
```

Comment:
Normally with test data like this, you can put it in a folder called `testdata` in the package and then read the contents during the test. Search for `testdata` folder and you'll see many examples.

Reply:
I am aware of testdata. But I don't think we need fixtures for this simple test.

Reply:
I would very much prefer putting this in a separate file, it makes the test a lot easier to read. 100+ lines of sshd config doesn't sit very well inside what would otherwise be a 40 lines Go test file.

Reply:
I think this is opinionated and creating a whole directory (at least to me) for a single sshdConfig test is a bit much. For a middle ground, I've removed most of the config and left just the header and 2 commented lines after the port.